src/main/java/org/opentripplanner/middleware/otp/OtpRequestProcessor.java

```diff
@@ -76,6 +78,15 @@ private static String proxy(Request request, spark.Response response) {
         return otpDispatcherResponse.responseBody;
     }
 
+    /**
+     * Get request path intended for OTP API by removing the proxy endpoint (/otp).
+     */
+    static String stripFirstOtpPathPrefix(String uri) {
+        // Only remove the first /otp proxy endpoint.
+        // The other one, if any, is part of the OTP router path on the OTP server.
+        return uri.replaceFirst(OTP_PROXY_ENDPOINT, "");
```

Comment:
I don't think this comment needs to exist about the first vs. other `/otp` path components. Keeping a comment here makes sense but it should be reworded because we don't necessarily anticipate /otp/otp request paths.

Reply:
I have deleted this methid, it is irrelevant now with the renaming of the config variable OTP_API_ROOT.